backend/src/main/java/de/trailmate/backend/repository/TourRepository.java

```diff
@@ -11,16 +11,28 @@
 import java.util.Map;
 
 
+@NoArgsConstructor
+@AllArgsConstructor
+@Data
 
 @Repository
 public class TourRepository {
 
     private Map<String, Tour> tourList = new HashMap<>();
```

Comment:
Watch out here: The Map should better not be initialised, for dependency injection.
```suggestion
    private Map<String, Tour> tourList;
```